### PR TITLE
add support for CRAN/METACRAN

### PR DESCRIPTION
**Pull request by wamserma**
example JSON URL:
http://crandb.r-pkg.org//devtools

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -4618,6 +4618,47 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// CRAN/METACRAN integration.
+camp.route(/^\/cran\/([^\/])\/([^\/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var info = match[1]; // either `v` or `l`
+  var pkg = match[2]; // eg, devtools
+  var format = match[3];
+  var url = 'http://crandb.r-pkg.org//' + pkg;
```
Comment (paulmelnikow) [inline server.js]: Should the `//` after the hostname just be `/`?

File: try.html [hunk 2]
```diff
@@ -538,6 +538,10 @@ <h3 id="version"> Version </h3>
   <td><img src='/cpan/v/Config-Augeas.svg' alt=''/></td>
   <td><code>https://img.shields.io/cpan/v/Config-Augeas.svg</code></td>
   </tr>
+  <tr><th data-keywords='R'> CRAN: </th>
+  <td><img src='/cran/v/devtools.svg' alt=''/></td>
+  <td><code>https://img.shields.io/cran/v/devtools.svg</code></td>
+  </tr>
```
Comment (paulmelnikow) [inline try.html]: The example badge is rendering like this.

screen shot 2017-03-29 at 10 39 53 pm

Could you fix that?

Could you also add an entry for `/cran/l`?

**Comment by paulmelnikow** [comment]
Thanks for this contribution! Would like to get it merged. Left a couple minor comments. Sorry for the delay!

File: try.html (hunk 2 above)
Comment (wamserma) [inline try.html]: I'll add the changes but I can't verify them beyond the CI tests as I don't have a matching dev-system here anymore.

File: server.js (hunk 1 above)
Comment (wamserma) [inline server.js]: Im pretty sure there was a reason for the double slash the time I made the PR. I just checked and either link works to get the JSON.

**Comment by wamserma** [comment]
Ok, also updated from upstream. The remining linting error seems unrelated to my PR, so it is take it or leave it now.

**Comment by paulmelnikow** [comment]
That merge of master didn't work. This branch is showing all the changes from those commits. If current master were merged in it should only show your changes. I'll take a closer look later today.

**Comment by wamserma** [comment]
Yep, recalling I forgot to pull the patch-branch branch after cloning the repo, so when I pulled from upstream/master my local (fresh) branch just got fast-forwarded and the subsequent pull on my patch-1 merged that onto master. I rebased and rebased against master, the repo should be clean again.
If there are any other issues, maybe just scrap this PR and copy the two changes manually.

**Comment by paulmelnikow** [comment]
Hmm, the API seems to have changed completely. I just pushed a fix to your branch, but it makes me wonder: is the API stable?

**Comment by wamserma** [comment]
It should be reasonably stable. API documentation is here: https://www.r-pkg.org/services#api -- in case someone wants to add support for even more badge types.

**Pull request merged**